includes/restoreMappings.js

```diff
@@ -17,7 +17,8 @@ const { BackupError } = require('./error.js');
 const debug = require('debug');
 
 const mappingDebug = debug('couchbackup:mappings');
-const RESUME_COMMENT = ' \t ' + ' ' + '   ' + '  \t' + '\t\t' + ' '; // Special sequence of whitespace to mark a resume
+const marker = '@cloudant/couchbackup:resume';
+const RESUME_COMMENT = `${JSON.stringify({ marker })}`; // Special marker for resumes
```

Comment:
Should say, I'm glad for this change. The spaces thing was cute and smart, but I had a feeling it's a bit _too_ smart and subtle.

Reply:
Yeah after some more discussion we agreed it wasn't gaining us anything, if the backup file line was valid then the resume marker will always be on its own line anyway. If it's not on its own line then the write wasn't complete (even if the JSON was complete the missing `\n` means we didn't write the corresponding log entry) so the marker actually preventing the line from parsing is ultimately a good thing.